cc c'est pau

Paupauuu opened the pull request.

Eponas committed:
navbar base

Paupauuu committed:
cc c'est pau

Eponas committed:
navbar base

Eponas reviewed (approved):
have to change : main (form and db)

Eponas committed:
Merge branch 'main' into navbar

Eponas committed:
Merge pull request #2 from Eponas/navbar

Navbar

Eponas committed:
navbar update change to stateful add corrections

Gridou87 committed:
Adding new pages (LoveCompatibility and AstralTheme) to the project
Adding French to the DatePicker

Gridou87 committed:
Coding of LoveCompatibilityPage

Gridou87 committed:
Fix bug in display in AstralThemePage because of CalendarDatePicker too big for the screen

Gridou87 committed:
Creation and coding of ProfilePage
Putting another icon in navbar to access new page
New images for icon in navbar and icons in ProfilePage

Gridou87 committed:
Forgot file pubspec.yaml in precedent commit and push (oups)

Eponas committed:
Merge pull request #5 from Eponas/lovecompa

All pages except MainPage

Eponas committed:
Home page with mock DB

Eponas committed:
Merge branch 'main' into mainpage

Eponas committed:
Merge pull request #6 from Eponas/mainpage

Home page with mock DB

Eponas committed:
Home page with mock DB

Eponas committed:
Correctif merge

Eponas committed:
Merge pull request #7 from Eponas/mainpage

Mainpage

Paupauuu committed:
rectifications DB

Eponas committed:
Merge branch 'main' into database

Eponas committed:
Merge pull request #8 from Eponas/database

hotfix DB.

Penser a mettre a jour sa branche avant de push merci <3

Eponas committed:
Hotfix

Gridou87 committed:
Calculate the love compatibility and displays it

Gridou87 committed:
Better display on ProfilePage (text is centered now)

Gridou87 committed:
Calculate astral theme (lunar sign and chinese sign)

Gridou87 committed:
Remove some comments

Gridou87 committed:
Fix database problem on my computer

Eponas committed:
Merge pull request #9 from Eponas/usebdd

Usebdd

Eponas committed:
Centering Title and final homePage

Gridou87 committed:
Using database to calculate solar sign and chinese sign in AstralThemePage

Gridou87 committed:
Using database to have the names of the sign in the two dropdowns and to calculate love compatibility between two signs

Gridou87 committed:
Putting some comments to the code and centering the title of each page

Eponas committed:
Merge pull request #10 from Eponas/ingrid

Adding the database to my pages
